### PR TITLE
Update typechain, other minor deps

### PR DESCRIPTION
**Pull request by jacque006**
Resolves https://github.com/thehubbleproject/hubble-contracts/issues/564

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,28 +33,28 @@
...
     "minimist": "^1.2.5",
     "prettier": "^1.19.1",
-    "prettier-plugin-solidity": "1.0.0-beta.10",
-    "solhint": "^3.3.4",
-    "ts-node": "^9.1.1",
-    "typechain": "^2.0.1",
-    "typescript": "^4.2.4"
+    "prettier-plugin-solidity": "1.0.0-beta.11",
+    "solhint": "^3.3.6",
+    "ts-node": "^10.0.0",
```
Comment (jacque006) [inline package.json:49]: No breaking changes on our end from what I can see.

File: package.json [hunk 2]
```diff
@@ -33,28 +33,28 @@
...
-    "hardhat": "^2.2.1",
+    "hardhat": "^2.3.2",
     "minimist": "^1.2.5",
     "prettier": "^1.19.1",
-    "prettier-plugin-solidity": "1.0.0-beta.10",
-    "solhint": "^3.3.4",
-    "ts-node": "^9.1.1",
-    "typechain": "^2.0.1",
-    "typescript": "^4.2.4"
+    "prettier-plugin-solidity": "1.0.0-beta.11",
```
Comment (jacque006) [inline package.json:47]: We will not be able to go higher on this version until https://github.com/thehubbleproject/hubble-contracts/issues/568

File: test/fast/accountTree.test.ts [hunk 3]
```diff
@@ -1,11 +1,18 @@
...
 import { Tree, Hasher } from "../../ts/tree";
 import { ethers } from "hardhat";
 import { assert } from "chai";
 import { randHex, randomLeaves } from "../../ts/utils";
 import { ZERO_BYTES32 } from "../../ts/constants";
 
+type UpdateBatchLeafs = Parameters<TestAccountTree["updateBatch"]>[0];
+type CheckInclusionTreeWitness = Parameters<
+    TestAccountTree["checkInclusion"]
+>[2];
```
Comment (jacque006) [inline test/fast/accountTree.test.ts:14]: This and the usage points below are the best way I could determine to allow compatibility with Solidity tuple types now being generated in TS land (uint156[2] => [BigNumberish, BigNumberish]).

Would be nice if we could verify in Node.js before making txn calls that the length of arrays is valid, but for now will have to rely on the txn failing.

File: ts/mcl.ts [hunk 4]
```diff
@@ -14,8 +14,8 @@ export type MessagePoint = mclG1;
 export type Signature = mclG1;
 export type PublicKey = mclG2;
 
-export type solG1 = [string, string];
-export type solG2 = [string, string, string, string];
+export type solG1 = [BigNumberish, BigNumberish];
+export type solG2 = [BigNumberish, BigNumberish, BigNumberish, BigNumberish];
```
Comment (jacque006) [inline ts/mcl.ts:18]: Based on my understanding of `mcl` and how these types are used in TS/JS code and at the contract level, this seems like a more appropriate type.

**Review by ChihChengLiang — APPROVED** [review]
Looks good to me 👍

**Pull request merged**